bin/resty

```diff
@@ -35,7 +35,7 @@ my @all_args = @ARGV;
 my (@http_includes, @main_includes);
 
 GetOptions("c=i",             \(my $conns_num),
-           "e=s",             \(my $src),
+           "e=s@",            \(my $src_a),
```

Comment:
I suggest use an array variable here, just like the `--http-include` option below.